src/cuttlefish.app.src

```diff
@@ -1,6 +1,6 @@
 {application,cuttlefish,
              [{description,"cuttlefish configuration abstraction"},
-              {vsn,"2.7.0"},
+              {vsn,"3.0.0"},
```

Comment:
If no API was broken by the patch, we don't need to bump the major number.

Reply:
The API wasn't but some things have changed: the timestamp formatting, and all the Lager-specific features are gone. I think we can bump the major version as it hasn't been done in many years.